helm/charts/hydra/templates/configmap.yaml

```diff
@@ -7,6 +7,6 @@ metadata:
 {{ include "hydra.labels" . | indent 4 }}
 data:
   "config.yaml": |
-  {{- with .Values.hydra.config -}}
-    {{- toYaml . | nindent 4 }}
-  {{- end -}}
+  {{- $config := set .Values.hydra.config "dsn" "redacted" -}}
```

Comment:
I think we could use [`unset`](https://github.com/Masterminds/sprig/blob/master/docs/dicts.md#unset) instead of set.